src/app/projects/project-list/project-list-item.component.ts

```diff
@@ -29,9 +29,13 @@ export class ProjectListItemComponent implements OnChanges {
 
   ngOnChanges(changes: NgChanges<ProjectListItemComponent>): void {
     if (changes.project && this.project) {
-      this.providerHostname = this.project.provider?.url
-        ? new URL(this.project.provider.url).hostname
-        : null;
+      try {
+        this.providerHostname = this.project.provider?.url
+          ? new URL(this.project.provider.url).hostname
+          : null;
+      } catch (e) {
+        this.providerHostname = null;
```

Comment:
@Pikabanga brings a fair point. Would rather see a [PrimeNG tag](https://www.primefaces.org/primeng/showcase/#/tag) with the tooltip of "Invalid provider URL: {error message}" and then setting `this.providerHostname = this.project.provider.url` directly.

Ex:

| NAME | GROUP | PROVIDER | STATUS | ACTIONS |
| --- | --- | --- | --- | --- |
| sample | default | <span title="Invalid provider URL: missing hostname">`[error]`</span> /some/invalid-url | *N/A* | ...

Reply:
Ah I wonder if there was a misunderstanding here @fredx30. I meant an `[error]` tag to the table itself, for when an invalid host name was inserted by other means, such as from an invalid importer path, instead of just hiding the invalid hostname.